controller/eks-cluster-config-handler.go

```diff
@@ -650,11 +650,16 @@ func (h *Handler) createOrGetServiceRole(ctx context.Context, config *eksv1.EKSC
 	if aws.ToString(config.Spec.ServiceRole) == "" {
 		logrus.Infof("Creating service role")
 
+		serviceRoleTemplate, err := templates.GetServiceRoleTemplate(config.Spec.Region)
+		if err != nil {
+			return "", err
```

Comment:
```suggestion
			return "", fmt.Errorf("error getting service role template: %v", err)
```

Reply:
updated this one and also the err message for other templates